src/backend/utils/poster.py

```diff
@@ -29,8 +27,10 @@ async def generate_poster(capture_id):
     # Combine frames into one image
     combined_frames = combine_frames(io_paths['input'])
 
+    template_path = "../../src/backend/utils/poster_template.png"
+
     if not os.path.exists(template_path):
-        return f"Could not find template at {template_path}"
+        print(f"Could not find template at {template_path}")
```

Comment:
@LJMU-JackMallo What happens here if there is no template? Is it just caught in the `except` block below?